### PR TITLE
Run new_gallery__transition_perf on Pixel 7 and Samsung A02

### PR DESCRIPTION
**Pull request by zanderso**

### CODE REVIEW

File: .ci.yaml [hunk 1]
```diff
@@ -50,6 +50,18 @@ platform_properties:
...
+  linux_pixel_7pro:
+    properties:
+      dependencies: >-
+        [
+          {"dependency": "android_sdk", "version": "version:33v6"},
+          {"dependency": "open_jdk", "version": "version:11"},
+          {"dependency": "curl", "version": "version:7.64.0"}
+        ]
+      os: Linux
+      device_type: "Pixel 7 Pro"
```
Comment (zanderso) [inline .ci.yaml:63]: Wondering if this should be "cheetah"?

Comment (zanderso) [reply .ci.yaml:63]: Yeah, I'm hoping to keep "Pixel 7 Pro"

File: .ci.yaml [hunk 2]
```diff
@@ -2326,6 +2329,17 @@ targets:
         ["devicelab", "android", "linux"]
       task_name: flutter_gallery__transition_perf_with_semantics
 
+  # MotoG4, Skia
+  - name: Linux_android new_gallery__transition_perf
```
Comment (yusuf-goog) [inline .ci.yaml:2333]: should this build be in this PR?

**Review by yusuf-goog — APPROVED** [review]
LGTM with one question

File: .ci.yaml (hunk 2 above)
Comment (zanderso) [reply .ci.yaml:2333]: I moved this existing build from another place in the file to be next to all the other `new_gallery__transition_perf` benchmarks.

File: .ci.yaml [hunk 3]
```diff
@@ -2346,6 +2383,30 @@ targets:
...
+    bringup: true
+    presubmit: false
+    timeout: 60
+    properties:
+      tags: >
+        ["devicelab", "android", "linux", "pixel", "7pro"]
+      task_name: new_gallery_impeller__transition_perf
+
+  # Samsung A02, Impeller (OpenGL)
+  - name: Linux_samsung_a02 new_gallery_impeller__transition_perf
```
Comment (keyonghan) [inline .ci.yaml:2398]: Just a heads up that we do not have any samsung a02 in the prod pool yet. What is the plan to add capacity in prod before enabling these tasks there? @yusuf-goog

Comment (yusuf-goog) [reply .ci.yaml:2398]: We are not enabling them in prod yet. Once zac is happy, i'll move some over to prod.

**Pull request merged**